Tests/BacktraceMetricsTest.swift

```diff
@@ -16,35 +16,31 @@ final class BacktraceMetricsTests: QuickSpec {
 
             let summedEventName = "view-changed"
             let uniqueEventName = "guid"
-
-            it("clears the summed event after enabling and sending") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-            }
-
-            it("can add and store summed event") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-
-                metrics.addSummedEvent(name: summedEventName)
-                expect { metrics.count }.to(equal(2))
-            }
-
-            it("can add and store unique event") {
-                let metrics = BacktraceMetrics(api: backtraceApi)
-                metrics.enable(settings: BacktraceMetricsSettings())
-
-                // Allow default events to be "sent" out
-                expect { metrics.count }.toEventually(equal(1), timeout: .seconds(5), pollInterval: .milliseconds(100))
-
-                metrics.addUniqueEvent(name: uniqueEventName)
-                expect { metrics.count }.to(equal(2))
+            
+            var metrics: BacktraceMetrics?
+            
+            context("Events operation") {
+                
+                beforeEach {
+                    metrics = BacktraceMetrics(api: backtraceApi)
+                    metrics?.enable(settings: BacktraceMetricsSettings())
```

Comment:
enable method should generate a summed + unique event. Are you sure we're testing the right thing? Maybe we should clear summed events if we want to test unique? I think its worth testing if the default summed +unique events are generated.

Reply:
all I did was cleanup by moving metrics init and enable to beforeEach.
Tests premises remain unchanged.
I added clearSummedEvents to test but it's not needed at all. I'll revert tests to the way they were.

Reply:
I think we're not testing the most important functionality, which is checking if the unique/summed event will be generated on the app startup. Maybe we can move cleanup method to each test and allow the test to decide what is the behavior under the test?